contributing-docs/testing/unit_tests.rst

```diff
@@ -738,6 +738,39 @@ You can also use fixture to create object that needs database just like this.
         conn = request.getfixturevalue(conn)
         ...
 
+Running tests with Database isolation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Running tests with DB isolation is a special case of tess that require ``internal-api`` component to be
+started in order to execute the tests. Only selected tests can be run with the database isolation
+(TODO: add the list) - they are simulating running untrusted components (dag file processor, triggerer,
```

Comment:
Is the todo something missed or something to be added in a follow-up later?

Reply:
Follow up later - we have no full list yet.